Python/pywarpx/picmi.py

```diff
@@ -923,15 +923,27 @@ class EmbeddedBoundary(picmistandard.base._ClassWithInit):
                  when the solver is electrostatic. Optional, defaults to 0.
      Parameters used in the expressions should be given as additional keyword arguments.
     """
-    def __init__(self, implicit_function, potential=None, **kw):
+    def __init__(self, implicit_function=None, stl_file=None, stl_scale=None, stl_center=None, stl_reverse_normal=None,
+                 potential=None, **kw):
         self.implicit_function = implicit_function
+        self.stl_file = stl_file
+
```

Comment:
Can you add a check to make sure that both `implicit_function` and `stl_file` are not set?

Reply:
Good idea thanks a lot! I've just added an assert